src/Utilities/Blaze.hpp

```diff
@@ -4,41 +4,80 @@
 /// \file
 /// Includes Blaze library with specific configs
 
+#include <blaze/math/CustomVector.h>
+#include <blaze/math/DenseVector.h>
+#include <blaze/math/GroupTag.h>
+
 #pragma once
 
 #ifdef __GNUC__
 #pragma GCC system_header
 #endif
```

Comment:
(optional) These lines should probably be deleted, but if that leads to warnings I'm fine with keeping them around for now